__tests__/auditTrail.test.ts

```diff
@@ -1,28 +1,23 @@
 import * as gqlTypes from '../src/typeDefs/gqlTypes.js'
-import { dbInstance } from '../src/firebaseDb.js'
-import { describe, it, expect} from 'vitest'
-import { createAuditLog } from '../src/services/auditLogService'
-import { generateRandomCreateSubmissionInput } from '../src/fakeData/fakeSubmissions'
+import * as dbSchema from '../src/typeDefs/dbSchema.js'
+import { describe, it, expect } from 'vitest'
+import { createAuditLogSQL } from '../src/services/auditLogServiceSupabase.js'
+import { generateRandomCreateSubmissionInput } from '../src/fakeData/fakeSubmissions.js'
 import { mapGqlEntityToDbEntity } from '../src/services/submissionService-pre-migration.js'
 
 describe('createAuditLog', () => {
     it('should create an audit log successfully', async () => {        
         const newSubmission: gqlTypes.CreateSubmissionInput = generateRandomCreateSubmissionInput()
-        const newAuditLog: gqlTypes.Submission = mapGqlEntityToDbEntity(newSubmission, '1')
+        const newAuditLog: dbSchema.SubmissionInsertRow = mapGqlEntityToDbEntity(newSubmission)
         
-        const auditLogResult = await dbInstance.runTransaction(async t => {
-            const result = await createAuditLog(
-                gqlTypes.ActionType.Create, 
-                gqlTypes.ObjectType.Submission, 
-                'user123', 
-                JSON.stringify(newAuditLog), 
-                null,
-                t
-            )
-
-            return result
-        })
-
-        expect(auditLogResult.isSuccesful).toBe(true)
+        await expect(
+            createAuditLogSQL({
+                actionType: 'CREATE',
+                objectType: 'Submission',
```

Comment:
Is there a reason we need to switch to hardcoding here? Is there a new `actionType` and `objectType` we could point to?